tests/Execution/integer-comparisons.java

```diff
@@ -0,0 +1,254 @@
+// RUN: javac %s -d %t
+// RUN: jllvm -Xenable-test-utils %t/Test.class | FileCheck %s
+
+class Test
+{
+    public static native void print(int i);
+
+    public static void main(String[] args)
+    {
+        int x = 5;
+        int y = -7;
+        int z = 0;
+
+        // ifne
+        //CHECK: 0
+        if (y == 0)
```

Comment:
Do you know whethere there is a reason this doesn't generate a `ificmpne` instead of a `ifne` or is this just `javac` being "weird"?
Also, for these tests that aren't required to generate `if<cmp>` instructions, would it suffice to just do `print(y == 0)` for example, instead of the conditional branch?

Reply:
Resolved offline, `0` forces the other comparison to be generated